src/main/java/com/sourcegraph/javagraph/AndroidCoreProject.java

```diff
@@ -0,0 +1,120 @@
+package com.sourcegraph.javagraph;
+
+import org.apache.commons.io.Charsets;
+import org.apache.maven.model.building.ModelBuildingException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.*;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.*;
+
+public class AndroidCoreProject implements Project {
```

Comment:
Can we add a test case for Android projects? Should be similar to the other test cases in the `testdata` directory.